extensions/podman/scripts/podman-download.spec.ts

```diff
@@ -63,6 +66,10 @@ class TestPodmanDownload extends PodmanDownload {
   public getShaCheck(): ShaCheck {
     return super.getShaCheck();
   }
+
+  public getDownloadAndCheck(): DownloadAndCheck {
+    return super.getDownloadAndCheck();
```

Comment:
Just curious, why make method from base class protected, then override it in ancestor and make it public? What is the idea here?

Reply:
to call the method directly from a unit test (but not being able to call it directly when we instantiate the class in the code base)

Reply:
(or to replace the method during test)